<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmHostInfo</class>
 <widget class="QWidget" name="frmHostInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>235</width>
    <height>140</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;宋体&quot;;</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLabel" name="labHostName">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>151</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>主机:测试主机1</string>
      </property>
     </widget>
     <widget class="QLabel" name="labStatus">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>70</y>
        <width>31</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>布防</string>
      </property>
     </widget>
     <widget class="myDefenceButton" name="hostOnLine" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="labOnLine">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>40</y>
        <width>31</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>在线</string>
      </property>
     </widget>
     <widget class="myDefenceButton" name="hostStatus" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btnBuFang">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>10</y>
        <width>61</width>
        <height>25</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>布防</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnCheFang">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>40</y>
        <width>61</width>
        <height>25</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>撤防</string>
      </property>
     </widget>
     <widget class="QLabel" name="labHighLow">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>100</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>高压(值:5.5kv)</string>
      </property>
     </widget>
     <widget class="myDefenceButton" name="hostHigh" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btnHigh">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>70</y>
        <width>61</width>
        <height>25</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>转高压</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnLow">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>100</y>
        <width>61</width>
        <height>25</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>转低压</string>
      </property>
     </widget>
     <widget class="myDefenceButton" name="hostAlarm" native="true">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>40</y>
        <width>61</width>
        <height>51</height>
       </rect>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>myDefenceButton</class>
   <extends>QWidget</extends>
   <header location="global">mydefencebutton.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
